### DISCUSSION TITLE
How to know which org current user belong to?

### DISCUSSION THREAD

**Discussion by nvcnvn** [zitadel/zitadel#8755] [Q&A] [answered]
Hello everyone!
I'm following this example setup https://zitadel.com/docs/guides/solution-scenarios/saas

The question is, how can I check which Org the current user belong to? Via backend API call or what kind of token?

Thank you!

**Comment by hifabienne** [comment, marked as answer]
if you include the resource owner scope in the authrequest, you will get the resource owner in the token e.g the organization.
urn:zitadel:iam:user:resourceowner
https://zitadel.com/docs/apis/openidoauth/scopes#reserved-scopes

Otherwise when you request the user on the api, you will also have the resource owner in the details of the user object
https://zitadel.com/docs/apis/resources/user_service_v2/user-service-get-user-by-id

**Reply by nvcnvn** [reply]
Thank you for quick support! Worked like a charm!
